packages/@lwc/synthetic-shadow/src/faux-shadow/shadow-root.ts

```diff
@@ -46,20 +46,34 @@ import { pathComposer } from '../3rdparty/polymer/path-composer';
 import { getInternalChildNodes, setNodeKey, setNodeOwnerKey } from './node';
 import { innerHTMLSetter } from '../env/element';
 import { getOwnerDocument } from '../shared/utils';
+import { adoptHostToSupportSlotting } from './slot';
 
 const { getHiddenField, setHiddenField, createFieldName } = fields;
 const ShadowRootResolverKey = '$shadowResolver$';
 const InternalSlot = createFieldName('shadowRecord', 'synthetic-shadow');
 const { createDocumentFragment } = document;
 
+export interface SlottedNodeRecord {
```

Comment:
we use a tuple here to keep the node and the slot name association, not sure how to declare that with arrays, so, using indexes with an object instead. any idea? maybe just make it an obj and put the proper names there.

Reply:
I think it's just `export type SlottedNodeRecord = [Node, string];`